test/core/002.phpt

```diff
@@ -50,14 +50,14 @@ var_dump($r->bar);
 ===DONE===
 --EXPECTF--
 ReflectionMethodEx::__construct
-%unicode|string%(26) "ReflectionFunctionAbstract"
-%unicode|string%(7) "getName"
-%unicode|string%(3) "xyz"
+string(26) "ReflectionFunctionAbstract"
```

Comment:
I was actually looking for 7.2 tests added around `object`, and figured that replacing them as per current state would be much easier